shared-libs/cht-datasource/test/local/place.spec.ts

```diff
@@ -87,99 +87,77 @@ describe('local place', () => {
       });
 
       it('returns null if the doc returned is not a place', async () => {
-        const place0 = { _id: 'place0', _rev: 'rev' };
-        const place1 = { _id: 'place1', _rev: 'rev' };
-        const place2 = { _id: 'place2', _rev: 'rev' };
-        getLineageDocsByIdInner.resolves([place0, place1, place2]);
+        const notPlace = { type: 'not-place', _id: 'place0', _rev: 'rev' };
+        const mockFunction = sinon.stub().resolves(notPlace);
+        mockFetchHydratedDoc.returns(mockFunction);
         isPlace.returns(false);
         settingsGetAll.returns(settings);
 
         const result = await Place.v1.getWithLineage(localContext)(identifier);
 
         expect(result).to.be.null;
-        expect(getLineageDocsByIdInner.calledOnceWithExactly(identifier.uuid)).to.be.true;
-        expect(isPlace.calledOnceWithExactly(settings, place0)).to.be.true;
+        expect(mockFetchHydratedDoc.calledOnceWithExactly(localContext.medicDb)).to.be.true;
+        expect(mockFunction.calledOnceWithExactly(identifier.uuid)).to.be.true;
+        expect(isPlace.calledOnceWithExactly(settings, notPlace)).to.be.true;
         expect(warn.calledOnceWithExactly(`Document [${identifier.uuid}] is not a valid place.`)).to.be.true;
         expect(debug.notCalled).to.be.true;
-        expect(getContactLineageInner.notCalled).to.be.true;
-        expect(getContactLineageOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
-        expect(getLineageDocsByIdOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
       });
 
       it('returns a place if no lineage is found', async () => {
-        const place = { _id: 'place0', _rev: 'rev' };
-        getLineageDocsByIdInner.resolves([place]);
+        const place = { type: 'place', _id: 'place0', _rev: 'rev' };
+        const mockFunction = sinon.stub().resolves(place);
+        mockFetchHydratedDoc.returns(mockFunction);
         isPlace.returns(true);
         settingsGetAll.returns(settings);
 
         const result = await Place.v1.getWithLineage(localContext)(identifier);
 
         expect(result).to.equal(place);
-        expect(getLineageDocsByIdInner.calledOnceWithExactly(identifier.uuid)).to.be.true;
+        expect(result).to.deep.equal(place);
+        expect(mockFetchHydratedDoc.calledOnceWithExactly(localContext.medicDb)).to.be.true;
+        expect(mockFunction.calledOnceWithExactly(identifier.uuid)).to.be.true;
         expect(isPlace.calledOnceWithExactly(settings, place)).to.be.true;
         expect(warn.notCalled).to.be.true;
-        expect(debug.calledOnceWithExactly(`No lineage places found for place [${identifier.uuid}].`)).to.be.true;
-        expect(getContactLineageInner.notCalled).to.be.true;
-        expect(getContactLineageOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
-        expect(getLineageDocsByIdOuter.calledOnceWithExactly(localContext.medicDb)).to.be.true;
       });
```

Comment:
```suggestion
```

suggestion: can just remove this test case since the logic does not exist anymore.